src/types/index.ts

```diff
@@ -40,6 +40,8 @@ export enum InstallSourceAndroid {
 }
 
 export interface ProductCommon {
+  type: 'subs' | 'sub' | 'inapp' | 'iap';
```

Comment:
We currently have 'iap' and 'sub' for iOS and
`inapp` and `subs' for [android which is the official type](https://developer.android.com/reference/com/android/billingclient/api/BillingClient.SkuType).

Looks like we can unify this.

Looks